dataprofiler/profilers/profiler_options.py

```diff
@@ -260,6 +267,53 @@ def _validate_helper(self, variable_path: str = "HistogramOption") -> list[str]:
                     "than 1, a string, or list of strings from the "
                     "following: {}.".format(variable_path, valid_methods)
                 )
+
+        if self.num_quantiles is not None and (
+            not isinstance(self.num_quantiles, int) or self.num_quantiles < 1
+        ):
+            errors.append(f"{variable_path}.num_quantiles must be a positive integer.")
+
+        return errors
+
+
+class HistogramOption(HistogramAndQuantilesOption):
```

Comment:
do we need to create an entire new option set or can we just in the above update the loader to use `HistogramAndQuantilesOption`